src/stores/game.js

```diff
@@ -13,22 +13,27 @@ function queenCount(player) {
   }
   return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
 }
-  
+
 const compareByRankThenSuit = (card1, card2) => {
-  return (card1.rank - card2.rank) || (card1.suit - card2.suit);
+  return card1.rank - card2.rank || card1.suit - card2.suit;
```

Comment:
I know this is a linter thing but imo this reads more clearly with the ().
```suggestion
  return (card1.rank - card2.rank) || (card1.suit - card2.suit);
```